test/fixtures/output/shell/curl/multipart-data.sh

```diff
@@ -1,4 +1,4 @@
 curl --request POST \
   --url http://mockbin.com/har \
   --header 'content-type: multipart/form-data; boundary=---011000010111000001101001' \
-  --form 'foo=Hello World'
+  --form foo=@hello.txt
```

Comment:
Just checking that this was intentional? I'm guessing the previous output was broken behavior and this is now fixed. 😄

Reply:
Yep, intentional. Since the `multipart-data` fixture in `test/fixtures/requests` has `params[0].fileName` for the parameter, we're now opting to supply that file directly to cURL instead of the contents of the file in `params[0].value`.